target/linux/mvebu/base-files/lib/upgrade/platform.sh

```diff
@@ -16,10 +16,7 @@ platform_do_upgrade() {
 	armada-385-linksys-caiman|armada-385-linksys-cobra|armada-385-linksys-rango|armada-385-linksys-shelby|armada-xp-linksys-mamba)
 		platform_do_upgrade_linksys "$ARGV"
 		;;
-	armada-388-clearfog-pro)
-		platform_do_upgrade_clearfog "$ARGV"
-		;;
-	armada-388-clearfog-base)
+	armada-388-clearfog|armada-388-clearfog-base|armada-388-clearfog-pro)
```

Comment:
There is no "armada-388-clearfog" anymore, it has been split into armada-388-clearfog-base and armada-388-clearfog-pro. So either drop armada-388-clearfog or use armada-388-clearfog-* as 02_network does.